README.org

```diff
@@ -271,6 +271,49 @@ A następnie należy podać odpowiednią linijkę z dobrze dobranym terminarzem.
 0 */2 * * * restic -r b2:bucketname:/path/to/repo backup /path/to/backed/up/data
 #+end_src
 
+* Przykład konfiguracji na NixOS
+NixOs to system operacyjny stworzony z myślą o reprodukcyjności. Cały system może zostać skonfigurowany korzystając z plików ~.nix~. W plikach tych deklaruje się jakie funkcje systemu mają być dostępne, a nix zadba o to, aby spełnione były wszystkie zależności (co do hash i wersji zależności). Zależności te przechowuje w ~Nixstore~ z nazwami jednoznacznie określającymi wesję-hash-nazwę np.
```

Comment:
```suggestion
NixOS to system operacyjny stworzony z myślą o reprodukowalności. Cały system może zostać skonfigurowany korzystając z plików ~.nix~. W plikach tych deklaruje się jakie funkcje systemu mają być dostępne, a nix zadba o to, aby spełnione były wszystkie zależności (co do hash i wersji zależności). Zależności te przechowuje w ~Nix store~ z nazwami jednoznacznie określającymi hash-nazwa-wersja np.
```

reprodukcyjność to rzecz powiązana z prokreacją, a nie tworzeniem oprogramowania